docs/unified-test-documentation/dasharo-stability/capsule-update.md

```diff
@@ -0,0 +1,340 @@
+
+**Expected result**
+
+1. The network boot option should remain enabled after the system reboots and
+   the capsule update is applied.
+
+---
+
+## CUP150.001 Capsule Update
+
+**Test description**
+
+This test verifies a successful capsule update and ensures the correct user
+experience (UX) during the update.
+
+Each step in this test case should be executed in accordance with the
+instructions given in our
+[Capsule Update documentation](../../guides/capsule-update.md).
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. Custom logo firmware image
+
+**Test setup**
+
+1. Ensure the DUT is connected and the UEFI shell is available.
+1. Proceed with the common test setup steps as described in the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Obtain and save the BIOS version as `${original_bios_version}`.
+1. Perform a capsule update using a valid capsule.
+1. Reboot the system and select the UEFI Shell boot option.
+1. After the system boots, verify the BIOS version and save it as
+   `${updated_bios_version}`.
+1. Ensure that the BIOS version has changed.
+1. Verify that the status contains `CapsuleMax` and does not contain
+   `CapsuleLast`.
+
+**Expected result**
+
+1. The BIOS version should change after the update.
+1. The status should contain `CapsuleMax` and should not contain `CapsuleLast`.
+
+---
+
+## CUP160.001 Verifying BIOS Settings Persistence After Update - PART 2
+
+**Test description**
+
+This test aims to verify the persistence of BIOS settings after a capsule
+update.
+
+Each step in this test case should be executed in accordance with the
+instructions given in our
+[Capsule Update documentation](../../guides/capsule-update.md).
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Ensure the DUT is connected and the UEFI shell is available.
+1. Proceed with the common test setup steps as described in the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Enter the Setup Menu.
+1. Navigate to the "Boot Maintenance Manager" menu.
+1. Check the value of the "Auto Boot Time-out" option.
+
+**Expected result**
+
+1. The "Auto Boot Time-out" option should retain the previously set value.
+
+---
+
+## CUP170.001 Verifying UUID (Ubuntu)
+
+**Test description**
+
+This test checks if the UUID persists after a capsule update.
+
+Each step in this test case should be executed in accordance with the
+instructions given in our
+[Capsule Update documentation](../../guides/capsule-update.md).
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu
+
+**Test setup**
+
+1. Proceed with the [Test cases common documentation](#test-cases-common-documentation)
+   section.
+
+**Test steps**
+
+1. Log in to Ubuntu.
+1. Check the current UUID.
+1. Perform the capsule update using a valid capsule.
```

Comment:
Here and in the next two test cases: we don't perform an update in these tests. It assumes that `CUP150.001` is run before that, and only that case performs the update (to save both time and flash life expectancy). Also, the current (original would be a better word, as at this point it doesn't have to be current anymore) UUID, serial number and hash of logo are obtained in suite setup. Here we just log into Ubuntu, get new value and compare it against the old one saved earlier.